src/Apps/Order/Routes/__tests__/Shipping.jest.tsx

```diff
@@ -1062,30 +1062,100 @@ describe("Shipping", () => {
       })
     })
 
-    // eslint-disable-next-line jest/no-focused-tests
-    describe.only("with address verification enabled", () => {
```

Comment:
Realized we suppressed all other tests with this `.only`. 🙈

Reply:
Did the same in [my other PR](https://github.com/artsy/force/pull/12652/files) and found some failing tests. Curious to try out after merging this in

Reply:
👍  I'll merge this in and will probably create some merge conflicts. Hopefully straightforward to resolve but let me know if can help!

Reply:
I must have left those in while I was debugging tests last week. I'm sorry for not cleaning those up...